spec-0001.md

```diff
@@ -133,15 +153,23 @@ skimage.filters.gaussian(...)  # Lazy load `gaussian` from
 skimage.filters.rank.mean_bilateral(...)  # Loaded once `rank` is accessed
 ```
 
-# Implementation
+One disadvantage is that erroneous or missing imports no longer fail immediately.
+During development and testing, the `EAGER_IMPORT` environment variable can be set to disable lazy loading, so that errors like these can be spotted.
 
-<!--
-Discuss how this would be implemented.
--->
+#### External libraries
+
+The `lazy.install_lazy` function is an alternative to setting up package internal imports.
+We also provide `lazy.load` so that projects can lazily import external libraries:
+
+```python
+linalg = lazy.load('scipy.linalg')  # `linalg` will only be loaded when accessed
+```
+
+# Implementation
 
-Currently, a test implementation lives in [this pull request to scikit-image](https://github.com/scikit-image/scikit-image/pull/5101)—specifically, inside of [`lazy.py`](https://github.com/scikit-image/scikit-image/blob/f5727872efec270d643dd2c281b6f245b03ff937/skimage/util/lazy.py).
+Currently, a work-in-progress implementation lives in [this pull request to scikit-image](https://github.com/scikit-image/scikit-image/pull/5101)—specifically, inside of `lazy.py`.
```

Comment:
Might be worth linking to napari and networkx PRs.

Reply:
- https://github.com/napari/napari/pull/2816
- https://github.com/networkx/networkx/pull/4909